core/templates/dev/head/css/oppia.css

```diff
@@ -3491,6 +3491,14 @@ md-card.preview-conversation-skin-supplemental-card {
     left: -o-calc(50% - -200px);
     left: calc(50% - -200px);
   }
+  .oppia-splash-button-heading {
+      margin-left: auto;
```

Comment:
Sorry, a small thing: could you please indent by 2 here, in order to be consistent with the rest of the CSS?